maat-court-data-api/src/test/java/gov/uk/courtdata/integration/eform/EFormIntegrationTest.java

```diff
@@ -30,11 +31,16 @@ class EFormIntegrationTest {
     private static final int USN = 12334455;
     private static final String TYPE = "CRM14";
     private static final int MAAT_REF = 12334455;
+    private static final String XML_DOC = "<formData xmlns=\\\"http://eforms.legalservices.gov.uk/lscservice\\\"></formData>";
```

Comment:
How about if we load the actual XML file and try to run the scenarios including data validation and logic etc?

Reply:
Most of our API's are used within our team and this specific one will be used by an external team (via our adaptor) so there may be changes to the schema so having more detail integration test would be great and helpful.